### PR TITLE
Sync boshrelease with latest cg-uaa merge

### PR DESCRIPTION
**Pull request by rogeruiz**
Tasks:
- [x] Update submodules
- [x] Upgrade versions in `config/blob.yml`
- [x] Update blobstore across environments
- [x] Clean up `config/blobs.yml` ( :thinking: )
- [x] Various `packages/` updates
- [x] Various `jobs/` updates

cc: @linuxbozo, @cnelson

### CODE REVIEW

**Comment by rogeruiz** [comment]
The `Add * changes` commits leveraged this script to perform a `diff` between release job files.

```
#!/bin/bash

set -eu

uaa_theirs=~/Developer/cloud.gov/uaa-release
uaa_ours=~/Developer/cloud.gov/cg-cf-release

# shellcheck disable=SC2045
for file in $(ls $uaa_ours/jobs/uaa/templates)
do
  file_name=$(echo "$file" | cut -d '/' -f 2)
  original_file="$uaa_theirs/jobs/uaa/templates/$file_name"
  if [ -a "$original_file" ]
  then
    vimdiff "$original_file" "$uaa_ours/jobs/uaa/templates/$file_name"
  fi
done
```

**Comment by LinuxBozo** [comment]
@rogeruiz so that script doesn't get lost, maybe make it so it only has to ask for theirs path, and commit to this repo?

File: packages/uaa/spec [hunk 1]
```diff
@@ -4,5 +4,5 @@ dependencies:
 files:
 - uaa/**/*
 - openjdk/jce_policy-8.zip
-- openjdk/openjdk-1.8.0_91-x86_64-mountainlion.tar.gz
-- openjdk/openjdk-1.8.0_91-x86_64-trusty.tar.gz
+- openjdk/openjdk-1.8.0_101-x86_64-mountainlion.tar.gz
+- openjdk/openjdk-1.8.0_101-x86_64-trusty.tar.gz
```
Comment (LinuxBozo) [inline packages/uaa/spec:8]: Given: https://gsa-tts.slack.com/archives/cloud-gov-atlas/p1481303821000750, I would expect more changes here.

**Review by LinuxBozo — CHANGES_REQUESTED** [review]

File: packages/uaa/spec (hunk 1 above)
Comment (rogeruiz) [reply packages/uaa/spec:8]: right on. I am now seeing a whole bunch of mess. 😞

**Comment by rogeruiz** [comment]
Would it be safe to assume that if a deprecated property was removed, that it now needs to be removed from our spec and possibly refactored in our templates?

**Comment by LinuxBozo** [comment]
@rogeruiz Bingo. I would expect updates to either cg-deploy manifest or secrets or both.

**Review by LinuxBozo — APPROVED** [review]

**Pull request merged**